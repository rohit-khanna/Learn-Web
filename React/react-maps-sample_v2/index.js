import React from "react";
import { render } from "react-dom";
import "bootstrap/dist/css/bootstrap.min.css";
import "./index.css";
import App from "./src/components/App";
import ConfigureStore from "./src/redux/configureStore";
import InitialState from "./src/redux/reducers/InitialState";
import { Provider as ReduxProvider } from "react-redux";

const store = new ConfigureStore(InitialState);

render(
  <ReduxProvider store={store}>
    <App />
  </ReduxProvider>,
  document.getElementById("app")
);
