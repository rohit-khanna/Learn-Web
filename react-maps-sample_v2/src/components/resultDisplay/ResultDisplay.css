.resultContainer {
  margin: 1.5rem 0;
}

.lbl-heading {
  font-weight: 600;
}
