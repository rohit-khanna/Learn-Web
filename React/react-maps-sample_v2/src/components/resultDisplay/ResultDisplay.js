import React from "react";
import "./ResultDisplay.css";

const ResultDisplay = ({ totalDistance, totalTime }) => {
  return (
    <div className="resultContainer">
      <div className="row">
        <div className="col-md-5 lbl-heading">Total Distance:</div>
        <div className="col-md-auto">{totalDistance}</div>
      </div>
      <div className="row">
        <div className="col-md-5 lbl-heading">Total Time:</div>
        <div className="col-md-auto">{totalTime}</div>
      </div>
    </div>
  );
};

export default ResultDisplay;
