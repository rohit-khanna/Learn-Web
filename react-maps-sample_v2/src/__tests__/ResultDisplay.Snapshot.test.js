import React from "react";
import ResultDisplay from "../components/resultDisplay";
import renderer from "react-test-renderer";

describe("<ResultDisplay/>", () => {
  it("matches the snapshot", () => {
    const tree = renderer.create(
      <ResultDisplay totalDistance={1000} totalTime={180} />
    );

    expect(tree).toMatchSnapshot();
  });
});
