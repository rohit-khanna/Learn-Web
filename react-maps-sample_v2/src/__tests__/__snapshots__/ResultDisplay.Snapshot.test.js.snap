// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ResultDisplay/> matches the snapshot 1`] = `
<div
  className="resultContainer"
>
  <div
    className="row"
  >
    <div
      className="col-md-5 lbl-heading"
    >
      Total Distance:
    </div>
    <div
      className="col-md-auto"
    >
      1000
    </div>
  </div>
  <div
    className="row"
  >
    <div
      className="col-md-5 lbl-heading"
    >
      Total Time:
    </div>
    <div
      className="col-md-auto"
    >
      180
    </div>
  </div>
</div>
`;
